source/blog/2019-02-18-ember-3-8-released.md

```diff
@@ -0,0 +1,133 @@
+
+
+Computed Property Volatility (3 of 5)
+
+`.volatile()` is a computed property modifier which makes a computed property recalculate every time it is accessed, instead of caching. It also prevents property notifications from ever occuring on the property, which is generally not the behavior that developers are after. Volatile properties are usually used to simulate the behavior of native getters, which means that they would otherwise behave like normal properties.
+
+To update, consider upgrading to native class syntax and using native getters directly instead. There's guide on how to do this in the [deprecations app](https://emberjs.com/deprecations/v3.x#toc_computed-property-volatile).
+
+Deprecate `@ember/object#aliasMethod` (4 of 5)
+
+`@ember/object#aliasMethod` is a little known and rarely used method that allows user's to add aliases to objects defined with EmberObject. The deprecation warning can be removed by refactoring it into having one function call the other directly. To see how to do this, please refer to the [deprecations app](https://emberjs.com/deprecations/v3.x#toc_object-alias-method)
+
+Component Manager Factory Function (5 of 5)
+
+`setComponentManager` no longer takes a string to associate the custom component class and the component manager. Instead you must pass a factory function that produces an instance of the component manager. For more information please refer to the [deprecations app](https://emberjs.com/deprecations/v3.x#toc_component-manager-string-lookup)
+
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.8
+
+#### New Features (0)
+
+No new features introduced in Ember Data 3.8.
+
+#### Deprecations (0)
+
+No new deprecations introduced in Ember Data 3.8.
+
+
+For more details on changes in Ember Data 3.8, please review the
+[Ember Data 3.8.0 release page](https://github.com/emberjs/data/releases/tag/v3.8.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npm install -g ember-cli-update
+ember-cli-update
+```
+
+This utility will help you to update your app or add-on to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and Ember Data, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or Ember Data by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 3.8
```

Comment:
sorry for the delay. these are the changes from 3.7.1 to v3.8.0 that will go into the changelog:

- [#8175](https://github.com/ember-cli/ember-cli/pull/8175) Do not watch `tests` directory when tests are disabled [@f1sherman](https://github.com/f1sherman)
- [#8309](https://github.com/ember-cli/ember-cli/pull/8309) blueprints/addon: Add contributing section [@knownasilya](https://github.com/knownasilya)
- [#8313](https://github.com/ember-cli/ember-cli/pull/8313) Update ember-ajax to v4.x [@maxwondercorn](https://github.com/maxwondercorn)
- [#8321](https://github.com/ember-cli/ember-cli/pull/8321) Cleanup repo README [@kennethlarsen/feature](https://github.com/kennethlarsen/feature)
- [#8052](https://github.com/ember-cli/ember-cli/pull/8052) Use non-greedy pattern for `{{content-for}}` [@mpirio](https://github.com/mpirio)
- [#8325](https://github.com/ember-cli/ember-cli/pull/8325) Add contributing guidelines section to Docs [@jamesgeorge007](https://github.com/jamesgeorge007)
- [#8326](https://github.com/ember-cli/ember-cli/pull/8326) package.json: Move `fixturify` to `devDependencies` [@Turbo87](https://github.com/Turbo87)
- [#8329](https://github.com/ember-cli/ember-cli/pull/8329) livereload-server: Fix logger output [@Turbo87](https://github.com/Turbo87)
- [#8328](https://github.com/ember-cli/ember-cli/pull/8328) tasks/server: Remove obsolete `exists-sync` dependency declaration [@Turbo87](https://github.com/Turbo87)
- [#8330](https://github.com/ember-cli/ember-cli/pull/8330) blueprints/app/gitignore: Ignore Yarn PnP files [@Turbo87](https://github.com/Turbo87)
- [#8331](https://github.com/ember-cli/ember-cli/pull/8331) Update `ember-cli-dependency-checker` to v3.1.0 [@Turbo87](https://github.com/Turbo87)
- [#8323](https://github.com/ember-cli/ember-cli/pull/8323) Dynamically fetch the `ember-source` version for MU blueprints [@ppcano](https://github.com/ppcano)
- [#8327](https://github.com/ember-cli/ember-cli/pull/8327) Improve `ember-cli` entry file [@Turbo87](https://github.com/Turbo87)
- [#8427](https://github.com/ember-cli/ember-cli/pull/8427) Fix install warnings for @babel/core [@jrjohnson](https://github.com/jrjohnson)

nothing substantial for app developers. worth noting though are the README changes for addon developers.

Reply:
ok, I think I was able to distill some prose from this. :+1: